source/agora/common/Block.d

```diff
@@ -54,4 +94,44 @@ public struct Block
 
     ///
     public Transaction tx;
+
+
+    /***************************************************************************
+
+        Implements hashing support
+
+        Params:
+            dg = hashing function
+
+    ***************************************************************************/
+
+    public void computeHash (scope HashDg dg) const nothrow @safe @nogc
```

Comment:
Why do you need this ?

Reply:
Right, I only need the hash of the previous header, which contains the transaction hash (merkle root later). Let me remove this.